firestore/query.go

```diff
@@ -1238,34 +1234,40 @@ type DocumentIterator struct {
 // always concrete types, and the fact that this one has two different implementations
 // is an internal detail.
 type docIterator interface {
-	next(opts ...RunOption) (*DocumentSnapshot, error)
-	getExplainMetrics() *ExplainMetrics
+	next() (*DocumentSnapshot, error)
+	getExplainMetrics() (*ExplainMetrics, error)
 	stop()
 }
 
+// GetExplainMetrics returns query explain metrics.
+// This is only present when ExplainOptions run option is provided in the query.
+func (it *DocumentIterator) GetExplainMetrics() (*ExplainMetrics, error) {
```

Comment:
Let's drop the `Get`. Always preferred when possible.

Reply:
Done